crates/rome_json_formatter/src/json/value/array_value.rs

```diff
@@ -1,10 +1,15 @@
 use crate::prelude::*;
+use rome_formatter::write;
 use rome_json_syntax::JsonArrayValue;
-use rome_rowan::AstNode;
+
 #[derive(Debug, Clone, Default)]
 pub(crate) struct FormatJsonArrayValue;
 impl FormatNodeRule<JsonArrayValue> for FormatJsonArrayValue {
     fn fmt_fields(&self, node: &JsonArrayValue, f: &mut JsonFormatter) -> FormatResult<()> {
-        format_verbatim_node(node.syntax()).fmt(f)
+        write!(f, [node.l_brack_token().format()])?;
+
+        write!(f, [group(&soft_block_indent(&node.elements().format()))])?;
+
+        write!(f, [node.r_brack_token().format()])
```

Comment:
Nit: You can merge the calls into a single `write` call
```suggestion
		let JsonArrayValueFields {
			l_brack_token,
			elements,
			r_brack_token
		} = node.as_fields();
		
        write!(f, [
        	l_brack_token.format(),
        	group(&soft_block_indent(&elements.format())),
        	r_brack_token.format()
      	])
```